eslint/babel-eslint-plugin/package.json

```diff
@@ -51,7 +51,7 @@
         },
         "peerDependencies": {
           "@babel/eslint-parser": "^7.11.0",
-          "eslint": "^8.9.0"
+          "eslint": "^8.9.0 || ^9.0.0"
```

Comment:
Can we drop ESLint 8 support in Babel 8? ESLint 9 should prevail at the time we release Babel 8.

Reply:
I saw some tweets about how a large part of ESLint's ecosystem is still not compatible with ESLint 9 so many projects cannot migrate, I'd wait a bit to see how the situation evolves.